config/application.rb

```diff
@@ -76,6 +76,7 @@ class Application < Rails::Application
         h[:page_id] = event.payload[:page_id] if event.payload[:page_id]
         h[:page_slug] = event.payload[:page_slug] if event.payload[:page_slug]
         h[:exception] = event.payload[:exception] if event.payload[:exception]
+        h[:session_id_hash] = event.payload[:session_hash] if event.payload[:session_hash]
```

Comment:
Should this be `session_id_hash`?

```suggestion
        h[:session_id_hash] = event.payload[:session_id_hash] if event.payload[:session_id_hash]
```

Reply:
it should!